platform/src/main/python/dlpx/virtualization/platform/_linked.py

```diff
@@ -112,6 +112,35 @@ def source_size_decorator(source_size_impl):
 
         return source_size_decorator
 
+    @staticmethod
+    def _from_protobuf_remote_mount(remote_mount):
+        return Mount(remote_environment=RemoteEnvironment.from_proto(
+            remote_mount.remote_environment),
+            mount_path=remote_mount.mount_path,
+            shared_path=remote_mount.shared_path)
+
+    @staticmethod
+    def _get_mounts_from_request(request):
+        staged_mount = request.staged_source.staged_mount
+        mounts = request.staged_source.mounts
+        if LinkedOperations._check_staged_mount(staged_mount) \
+                and mounts and len(mounts) > 0:
+            raise PluginRuntimeError(
+                'Either staged_mount or mounts can be present for staging source. '
+                'Found both staged_mount and mounts.')
+
+        if mounts and len(mounts) > 0:
```

Comment:
should we say this `if mounts and len(mounts) > 1`?
As anyway `shared_path` won't make sense and won't work if a single mount has been provided?

Reply:
As discussed, from mount specification we do pass only mounts when plugin provides more than 1 mounts. But as it is part of request, we just check if mounts or staged_mount is present in the request sent by Delphix and set it accordingly to staged_source.